interview-questions.md

```diff
@@ -8,62 +8,62 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-  Your answer:
+  Your answer: A parameter is a variable that is defined to the function, the argument is the actual data that is assigned to the parameter.
 
-  Researched answer:
+  Researched answer: A parameter is the variable inside the parentheses in the function definition. An argument is the value given to the function to run through.
 
 
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-  Your answer:
+  Your answer: The .map() built in predefined parameters take a function as an argument, acts on the array, iterates through the array, and returns the array with the result of the argument.
 
-  Researched answer:
+  Researched answer: The built in method .map() takes predefined parameters. The predefined parameter that cannot change is the length of the array returned. The length of the array will always be the same as the array referenced and each element will be run through the function. The predefined parameter that can change is the array of the current element and value passed through the function.
```

Comment:
The parameter(s) mentioned in this case is currentValue, index, and array where currentValue is required and index and array are optional.  Ex: `something.map(value => value * 5)`